OpenRA.Mods.Common/Traits/Mobile.cs

```diff
@@ -866,7 +866,11 @@ void IDeathActorInitModifier.ModifyDeathActorInit(Actor self, TypeDictionary ini
 		void INotifyBecomingIdle.OnBecomingIdle(Actor self)
 		{
 			if (self.Location.Layer == 0)
+			{
+				if (!Locomotor.CanStayInCell(self.Location))
```

Comment:
Can we please add a comment here like
```
// Make sure that units aren't left idling in a transit-only cell
// HACK: activities should be making sure that this can't happen in the first place!
```
?

Reply:
Can do